src/suppzern.jl

```diff
@@ -60,12 +60,24 @@ function Zcoefs2phi(Zcoefs, Zval)
     return phi
 end
 
+function construct_Zernimg(Zcoefs, img, initial_param::InitialParam, defocus::Bool = false)
+    Hz, Zval, H = construct_Zernmat(initial_param, size(img))
```

Comment:
What about passing `defocus` into `construct_Zernmat`? It seems a little awkward to return both `Hz` and `H`, and then use `H` to replace `Hz` if `defocus==true`.